src/Components/LabSwap/LabSwap.tsx

```diff
@@ -79,9 +78,20 @@ export const LabSwap = () => {
           if (btn !== null) btn.innerHTML = "Failed";
           alert(resp.err);
         }
-      } else if (btn !== null){
-         btn.innerHTML = 'Done!';
+      } else if (btn !== null) {
+         btn.innerHTML = 'Updating...';
       }
+    }).then(() => {
+      API.fetchUserTrades(user.user?.person_id).then(resp => {
+        if (btn !== null) btn.innerHTML = "Done!";
+        setTimeout(() => {
+          setUserTrades(resp);
+          selectedTradeBlocksState[1]({
+            offered: null,
+            requested: null
+          });
+        }, 1500);
+      });
```

Comment:
Updates the trades after the button is pressed. We add a short timeout here after the data gets back to prevent a jarring transition